pinot-plugins/pinot-minion-tasks/pinot-minion-builtin-tasks/src/main/java/org/apache/pinot/plugin/minion/tasks/segmentgenerationandpush/SegmentGenerationAndPushTaskExecutor.java

```diff
@@ -287,6 +287,9 @@ protected SegmentGenerationTaskSpec generateTaskSpec(Map<String, String> taskCon
       recordReaderSpec.setDataFormat(taskConfigs.get(BatchConfigProperties.INPUT_FORMAT));
       recordReaderSpec.setClassName(taskConfigs.get(BatchConfigProperties.RECORD_READER_CLASS));
       recordReaderSpec.setConfigClassName(taskConfigs.get(BatchConfigProperties.RECORD_READER_CONFIG_CLASS));
+      recordReaderSpec.setConfigs(
```

Comment:
Have we set up the code style? If not, let's set up the pinot style in IDE and apply the code style change.

Reply:
Fixed in bb90840fffa9e1f3d2f03af2f8270127bfe91787